src/common/convolution_pd.hpp

```diff
@@ -233,8 +234,16 @@ struct convolution_pd_t : public primitive_desc_t {
                         || invariant_dst_md()->data_type == dst_dt)
                 && (acc_dt == data_type::undef
                         || desc_.accum_data_type == acc_dt);
-        if (with_bias() && bia_dt != data_type::undef)
+        if (with_bias() && bia_dt != data_type::undef) {
+#ifdef __aarch64__
+            // ACL only supports s32 bias for quantization. Therefore internally
+            // we convert from f32 to s32. So here the types doesn't match.
+            if (utils::one_of(
+                        dst_dt, data_type_t::dnnl_s8, data_type_t::dnnl_u8))
+                return ok;
+#endif
             ok = ok && invariant_bia_md()->data_type == bia_dt;
+        }
```

Comment:
Common checks usually cover general primitive semantics.
Since this check is specific to ACL, please put this check in acl_convolution implementation.

Reply:
Done!